runelite-client/src/main/java/net/runelite/client/plugins/npchighlight/NpcSceneOverlay.java

```diff
@@ -147,8 +147,12 @@ private void renderNpcRespawn(final MemorizedNpc npc, final Graphics2D graphics)
 		}
 	}
 
-	private void renderNpcOverlay(Graphics2D graphics, NPC actor, Color color)
+	public void renderNpcOverlay(Graphics2D graphics, NPC actor, Color color)
```

Comment:
why public?

Reply:
This was how I grabbed the Interacting colour in config, there might be a better way to do this